src/ResourceManager/Kusto/Commands.Kusto/Commands/GetAzureRmKustoCluster .cs

```diff
@@ -0,0 +1,74 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Kusto.Models;
+using Microsoft.Azure.Commands.Kusto.Utilities;
+using Microsoft.Azure.Management.Kusto;
+using Microsoft.Azure.Management.Kusto.Models;
+
+namespace Microsoft.Azure.Commands.Kusto
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmKustoCluster", DefaultParameterSetName = ParameterSet),
+        OutputType(typeof(List<PSKustoCluster>))]
```

Comment:
Please have the cmdlet output an array instead of a List. PowerShell prefers using arrays.